warehouse/models/mart/gtfs/_mart_gtfs_fcts.yml

```diff
@@ -1163,3 +1163,56 @@ models:
         description: Timestamp of first service alert header referencing this trip.
       - name: sa_max_header_timestamp
         description: Timestamp of last service alert header referencing this trip.
+
+
+  - name: fct_service_alerts_trip_summaries
```

Comment:
Can we add some documentation about the columns / what exactly is being summarized in this and the other two?